src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -471,15 +471,15 @@ public void SetRigiditySliderTooltip(float rigidity)
         var healthChangeLabel = GetNode<Label>(RigiditySliderTooltipHealthLabelPath);
         var mobilityChangeLabel = GetNode<Label>(RigiditySliderTooltipSpeedLabelPath);
 
-        float healthChange = rigidity * Constants.MEMBRANE_RIGIDITY_HITPOINTS_MODIFIER;
-        float mobilityChange = -1 * rigidity * Constants.MEMBRANE_RIGIDITY_MOBILITY_MODIFIER;
+        string healthChange = (rigidity * Constants.MEMBRANE_RIGIDITY_HITPOINTS_MODIFIER).ToString("0.##", CultureInfo.CurrentCulture);
+        string mobilityChange = (-1 * rigidity * Constants.MEMBRANE_RIGIDITY_MOBILITY_MODIFIER).ToString("0.##", CultureInfo.CurrentCulture);
 
-        healthChangeLabel.Text = ((healthChange > 0) ? "+" : string.Empty)
+        healthChangeLabel.Text = ((Convert.ToDouble(healthChange, CultureInfo.CurrentCulture) > 0) ? "+" : string.Empty)
             + healthChange.ToString(CultureInfo.CurrentCulture);
```

Comment:
add the format specifier to this ToString, also use "F3" as the specifier as I've used that elsewhere for consistency.